bridge_adaptivity/module/static/module/js/collectionform.js

```diff
@@ -0,0 +1,20 @@
+
+var update_collection_form = function() {
+    var clFormUrl = $("#link-objects-form-update").data("collection_url");
+    var cl = $("select[name='collection_group-collection'] option:selected").val();
+    console.log(clFormUrl);
+    $.get(clFormUrl, {
+        collection_id: cl,
+      }, function(response) {
+        console.log(response);
+        $("div.collection_form").html(response);
+    })
+};
+
+
+$(document).ready(function() {
+
+    $("form select[name=collection_group-collection]").on("change", function() {
+      update_collection_form();
+    });
+});
```

Comment:
Missing a new line at the end of the file.

Reply:
Done